Basic-Car-Maintenance/Shared/MainView/Views/MainTabView.swift

```diff
@@ -83,6 +106,65 @@ struct MainTabView: View {
     }
 }
 
+extension MainTabView {
+    private func tabItem(for selection: TabSelection) -> some View {
+        Label(selection.label, systemImage: selection.image)
+    }
+    
+    /// Save screen content for specific selection
+    /// - Parameter selection: tab selection enum value
+    @ViewBuilder
+    private func selectionContent(for selection: TabSelection) -> some View {
+        switch selection {
+        case .dashboard:
+            DashboardView(userUID: authenticationViewModel.user?.uid)
+        case .odometer:
+            OdometerView(userUID: authenticationViewModel.user?.uid)
+        case .settings:
+            SettingsView(authenticationViewModel: authenticationViewModel)
+        }
+    }
+    
+    /// Returns last column of NavigationSplitView - detail content
+    @ViewBuilder
+    private func splitViewDetailContent() -> some View {
+        if let tabSelection = selectedTabId {
+            selectionContent(for: tabSelection)
+                .tag(tabSelection)
+        } else {
+            Text("Unexpected error occured.")
+        }
+    }
+    
+    /// Returns NavigationSplitView navigation
+    /// primarily used on iPad and Mac devices
+    @ViewBuilder
+    private func navigationSplitView() -> some View {
+        NavigationSplitView(columnVisibility: $columnVisibility) {
+            List(TabSelection.allCases, selection: $selectedTabId) { tabSelection in
+                Label(tabSelection.label, systemImage: tabSelection.image)
+            }
+            .navigationTitle("Sidebar")
+        } detail: {
+            splitViewDetailContent()
+        }
+    }
+    
+    /// Returns TabView navigation
+    /// primarily used on iPhone devices
+    @ViewBuilder func tabView() -> some View {
```

Comment:
```suggestion
    /// Primarily used on iPhone devices
    /// - Returns: `TabView` navigation
    @ViewBuilder func tabView() -> some View {
```

Changing the formatting of the documentation comment